### ISSUE TITLE
[FEATURE REQUEST] Backup/Restore - Firebird 3 -SKIP(_DATA) | Firebird 4 -INCLUDE(_DATA)/-SKIP(_DATA)

### ISSUE THREAD

**Issue by nLeonardt95** [mariuz/flamerobin#212]
Hello, it would be nice if these options for backup/restore could also be integrated into Flamerobin.

Since Firebird 3
[https://firebirdsql.org/file/documentation/release_notes/html/en/3_0/bk02ch11s03.html#d0e19573](url)

Since Firebird 4
[https://firebirdsql.org/file/documentation/release_notes/html/en/4_0/rlsnotes40.html#rnfb40-util-gbak-include-data](url)

**Comment by luronumen** [comment]
Hi @nLeonardt95 

FlameRobin already have options to Backup/Restore databases.

Please let us know if it is enough for you. 

BACKUP
![BACKUP](https://user-images.githubusercontent.com/8019466/122755031-c8133380-d26a-11eb-8f6f-f6a77e82a654.png)

RESTORE
![RESTORE](https://user-images.githubusercontent.com/8019466/122755038-ca758d80-d26a-11eb-936d-e30e842003d8.png)

**Comment by nLeonardt95** [comment]
Yes, I know that, but I mean the Skip_data and include_data option.

**Comment by Jdochoa** [comment]
Hi @nLeonardt95 ,

This is part of the FB3 / FB4 updates, please be patient. 

./jo

**Comment by mariuz** [comment]
This feature was implemented in **October 2022**.

Pull Request [**#282**](https://github.com/mariuz/flamerobin/pull/282), titled *"Update the backup/restore options #212"*, specifically addressed this issue by adding the requested `gbak` features for Firebird 3 and 4.

Key details from the implementation:
- **Backup**: Added options for Firebird 3+ (`-SKIP_DATA`) and Firebird 4+ (`-INCLUDE_DATA`).
- **Restore**: Added relevant options for the newer Firebird versions.
- **Reference**: The PR explicitly mentions it fixes issue #212.

The PR was merged by you on October 16, 2022. It looks like the issue was just never officially closed.
